ibldev_env.yml

```diff
@@ -0,0 +1,68 @@
+name: ibldev
+channels:
+  - defaults
+  - conda-forge
+  - conda-canary
+dependencies:
+  - asn1crypto=0.24.0=py37_1003
+  - blas=1.0=mkl
+  - ca-certificates=2019.6.16=hecc5488_0
+  - certifi=2019.6.16=py37_1
+  - cffi=1.12.3=py37hb32ad35_0
+  - chardet=3.0.4=py37_1003
+  - colorama=0.4.1=py_0
+  - colorlog=4.0.2=py37_1000
+  - cryptography=2.7=py37hb32ad35_0
+  - cycler=0.10.0=py_1
+  - entrypoints=0.3=py37_1000
+  - flake8=3.7.8=py37_1
+  - freetype=2.10.0=h563cfd7_1
+  - globus-sdk=1.8.0=py_0
+  - icc_rt=2019.0.0=h0cc432a_1
+  - icu=64.2=he025d50_0
+  - idna=2.8=py37_1000
+  - intel-openmp=2019.4=245
+  - jpeg=9c=hfa6e2cd_1001
+  - kiwisolver=1.1.0=py37he980bc4_0
+  - libblas=3.8.0=12_mkl
+  - libcblas=3.8.0=12_mkl
+  - liblapack=3.8.0=12_mkl
+  - libpng=1.6.37=h7602738_0
+  - matplotlib=3.1.1=py37_1
+  - matplotlib-base=3.1.1=py37h2852a4a_1
+  - mccabe=0.6.1=py_1
+  - mkl=2019.4=245
+  - mkl-service=2.1.0=py37hfa6e2cd_0
+  - numpy=1.17.0=py37hc71023c_0
+  - openssl=1.1.1c=hfa6e2cd_0
+  - pandas=0.25.0=py37he350917_0
+  - patsy=0.5.1=py_0
+  - pip=19.2.2=py37_0
+  - pycodestyle=2.5.0=py_0
+  - pycparser=2.19=py37_1
+  - pyflakes=2.1.1=py_0
+  - pyjwt=1.7.1=py_0
+  - pyopenssl=19.0.0=py37_0
+  - pyparsing=2.4.2=py_0
+  - pyqt=5.9.2=py37h6538335_2
+  - pysocks=1.7.0=py37_0
+  - python=3.7.3=h510b542_1
```

Comment:
is python>=3.7 necessary for ibllib? If not, I can change this. I noticed the CI runs on python=3.6.7.